docs/content/testing/running-tests.md

```diff
@@ -0,0 +1,56 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Running Tests
+
+You can run tests from the command line, or using the Unity Editor, through
+ the [Unity Test Runner window](https://docs.unity3d.com/Manual/testing-editortestsrunner.html).
+
+## 1. Prepare Workspace
+
+You need to do this once after you check out the repository, or if anything in
+ the project setup has changed.
+
+Open the repository directory in your terminal, for example using Git Bash.
+
+```bash
+./prepare-workspace.sh
+```
+
+## 2. (Optional) Run All Tests From The Command Line
+
+From the repository directory:
+
+```bash
+./ci/test.sh
```

Comment:
`build-test.sh`, running `test.sh` by itself will break if nothing is built.

Reply:
I'm thinking of adding `build.sh` as part of preparing the workspace instead

Reply:
Done, PTAL :)

Reply:
I didn't add `build-test.sh` because if they do this repeatedly it'll rebuild unnecessarily

Reply:
Equally, if you make changes and run `test.sh` without building the changes won't be reflected in the test. 

Realistically, you probably won't run `build-test.sh` yourself, you'd use the GUI tools in VS or Unity to run the tests.

Reply:
If you make code changes in the `unity` directory it will be picked up by the editor when it's running the tests.

For modifications to tools you will need to still build them I am guessing, but in most cases you should not need to use `build-test.sh`